internal/service/storage.go

```diff
@@ -103,7 +106,7 @@ func NewStorage(cfg StorageConfig) (*Storage, error) {
 
 	log.Debug().Interface("cluster-map", cm).Msg("Cluster map loaded")
 
-	if cm.IsValid(s.remoteConfigurationPath) {
+	if cm.IsValid(s.remoteConfigurationPath, StableVersion) {
```

Comment:
don't we need to also validate the canary version?

Reply:
We definitely need to, I just wanted to cover this in [CCXDEV-13091](https://issues.redhat.com/browse/CCXDEV-13091) - invalid canary version does not hurt the service as it functions now, so I didn't include it here.